internal/testclient/testeventsquery/client.go

```diff
@@ -11,38 +10,29 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 
-	"pocket/internal/mocks/mockclient"
-	"pocket/internal/testclient"
-	"pocket/pkg/client"
-	eventsquery "pocket/pkg/client/events_query"
-	"pocket/pkg/either"
-	"pocket/pkg/observable/channel"
+	"github.com/pokt-network/poktroll/internal/mocks/mockclient"
+	"github.com/pokt-network/poktroll/internal/testclient"
+	"github.com/pokt-network/poktroll/pkg/client"
+	eventsquery "github.com/pokt-network/poktroll/pkg/client/events_query"
+	"github.com/pokt-network/poktroll/pkg/either"
+	"github.com/pokt-network/poktroll/pkg/observable/channel"
 )
 
-// NewLocalnetClient creates and returns a new events query client that connects to the
-// localnet sequencer. It leverages the NewEventsQueryClient from the eventsquery package.
-//
-// Parameters:
-// t: Testing object which is passed for marking helper.
-// opts: Any additional options to configure the events query client.
-//
-// Returns: An instance of EventsQueryClient which connects to the localnet sequencer.
+// NewLocalnetClient creates and returns a new events query client that configured
+// for use with the localnet sequencer. Any options provided are applied to the client.
 func NewLocalnetClient(t *testing.T, opts ...client.EventsQueryClientOption) client.EventsQueryClient {
 	t.Helper()
 
 	return eventsquery.NewEventsQueryClient(testclient.CometLocalWebsocketURL, opts...)
 }
 
-// NewOneTimeEventsQuery creates a mock of the EventsQueryClient which expects a single call to the
-// EventsBytes method. It returns a mock client that emits event bytes to the provided publish channel.
-//
-// Parameters:
-// ctx: Context object to define the context for the operation.
-// t: Testing object.
-// query: The query string for which event bytes are fetched.
-// publishCh: Channel to which the event bytes are published.
-//
-// Returns: A mock instance of EventsQueryClient which behaves as described.
+// NewOneTimeEventsQuery creates a mock of the EventsQueryClient which expects
+// a single call to the EventsBytes method. It returns a mock client whose event
+// bytes method always  constructs a new observable. query is the query string
+// for which event bytes subscription is expected to be for.
+// The caller can simulate blockchain events by sending on publishCh, the value
```

Comment:
This part is awesome:
> // The caller can simulate blockchain events by sending on publishCh.

This part is less awesome:
> // the value
> // of which is set to the publish channel of the events bytes observable publish
> // channel.

I KIND of see what you were intending, but consider deleting it since I think it might be extra unnecessary context. Ditto below